src/Nethermind/Nethermind.Runner/Ethereum/Steps/InitializeBlockchainAuRa.cs

```diff
@@ -19,6 +19,7 @@
 using System.Linq;
```

Comment:
it would be great if the AuRa steps were kept separately in the AuRa projects instead of the EthereumRunner

this way AuRa could come as a plugin too

Reply:
I have mixed feelings. Would have to move all base steps somewhere else too as AuraSteps are based on them. Its not a priority now and would result in quite big refactoring.

Reply:
Agreed, not high priority but runner grows so please keep in the back of your mind. Similar notion as with NDM at the beginning. Over time it may grow to a crazy size.